syntax = "proto3";

package message;

service VarifyService {
  rpc GetVarifyCode (GetVarifyReq) returns (GetVarifyRsp) {}
}

message GetVarifyReq {
  string email = 1;
  int32 module = 2;
}

message GetVarifyRsp {
  int32 error = 1;
  string email = 2;
  string code = 3;
}